ceph_installer/tests/test_util.py

```diff
@@ -95,6 +95,26 @@ def test_redhat_storage_is_false(self):
         assert result == {'ceph_stable': True}
 
 
+class TestGetOSDConfigureExtraVars(object):
+
+    def setup(self):
+        self.data = dict(
+            host="node1",
+            fsid="1720107309134",
+            devices=['/dev/sdb'],
+            monitors=[{"host": "mon1.host", "interface": "eth1"}],
+            journal_devices=["/dev/sdc"],
+            journal_size=100,
+            public_network="0.0.0.0/24",
+        )
+
+    def test_journal_devices(self):
+        result = util.get_osd_configure_extra_vars(self.data)
+        assert "raw_multi_journal" in result
```

Comment:
in this case you should do one test per assert I think. The reason for this is because if say, all three of these asserts are failing (although unlikely) you will not see 3 failures, you will only get one failure at a time.

Ideally you want the test method name to match the actual test case. It is easier for granular tests to match (and harder if they don't).

So for `raw_multi_journal` it could be:

    def test_raw_multi_journal_is_set(self):
        result = util.get_osd_configure_extra_vars(self.data)
        assert result['raw_multi_journal'] is True